:py:mod:`pdstools.utils.errors`
===============================

.. py:module:: pdstools.utils.errors


Module Contents
---------------

.. py:exception:: NotApplicableError

   Bases: :py:obj:`ValueError`

   Inappropriate argument value (of correct type).


.. py:exception:: NotEagerError(operationType=None, defaultmsg='This operation is only possible in eager mode.')

   Bases: :py:obj:`ValueError`

   Operation only possible in eager mode.


